### PR TITLE
Input width

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

**Comment by ScottDowne** [comment]
@Pomax this is fixing https://github.com/MozillaFoundation/Advocacy/issues/1042

I found this was also an issue on mobile, but of a slightly different nature.

Now, we should default to the placeholder width, unless it's quite small, then we default to a set min-width.

For mobile, we use rems, and it should fit.

Testing /fr/ seems to be the best as it's the longest placeholder text.

I also matched the styling of the default placeholder to match the custom one.

I also added autoComplete="off" to ensure a soft refresh clears the input, otherwise we get some visual bugs.

Thoughts?

**Comment by TheoChevalier** [comment]
If this helps, I’m okay to switch French to `Saisissez votre n° de téléphone`, making German the longest (32 chars)

**Comment by ScottDowne** [comment]
@TheoChevalier whatever makes the most sense for you. I think this patch handles the length as is.

File: src/components/call-tool.js [hunk 1]
```diff
@@ -77,9 +77,36 @@ module.exports = React.createClass({
...
+
+    // This state is for when the input has been cleared,
+    // completely empty, so no county prefix.
+    if (!this.state.number) {
+      placeholderContainer = (
+        <span className="placeholder-container">
+          <span className="placeholder-width">{placeholder}&nbsp;</span>
+        </span>
+      );
+    // This state is for when the input is not empty and not default.
```
Comment (Pomax) [inline src/components/call-tool.js]: The comment placement here is a bit confusing, would this work for you?
```
}
// This state is for [...]
else if (....) {
  ..
}
```

Comment (ScottDowne) [reply src/components/call-tool.js]: That works for me.

**Pull request merged**